bin/sdc-thoth-update-all

```diff
@@ -72,8 +72,6 @@ echo "Downloading thoth to compute nodes ..."
 sdc-oneachnode -a -g $staged -d /var/tmp 1>&2
 echo "Installing thoth on compute nodes ..."
 sdc-oneachnode -a "cd / ; gzcat $tarball | tar xf - ; rm $tarball" 1>&2
-sdc-oneachnode -c -X -g /opt/custom/thoth/bin/thoth -d /opt/custom/thoth/bin/
-sdc-oneachnode -a 'chmod +x /opt/custom/thoth/bin/thoth'
```

Comment:
We need the thoth command to be available on all CNs for `thoth upload` and `thoth object`.

Reply:
This is still happening, this is an unnecessary additional change